en/page-breaking.saty

```diff
@@ -0,0 +1,288 @@
+    without returning contents to output PDF.
+    The reason was what causing \it{side-effect} at page-breaking not using return-value,
+    and you can not do it\footnote{
+      There is no reason you cannot do it.
+      If needed, it is sounds good to change the specification what imcompatible in future;
+      \it{\codem{page-break} itself processes a page-breaking}.}.}
+
+  >
++section{primitves related page-breaking}<
+    +p{
+      There are some primitives to process related page-breaking.
+      Let me introduce them here.
+
+      \val-explain?:(`clear-page`)({clear-page})%
+        (Type.(block-boxes)) {
+          Page break in force.
+          In case inserted it at a target page unbreakable, 
+          it is equivalent as 
+          \codem{block-nil}.
+        }%
+
+      \val-explain?:(`add-footnote`)({add-footnote})%
+        (Type.(block-boxes --> inline-boxes)) {
+          
+          \codem{add-footnote \meta{bb}} inserts block-boxes \codem{\meta{bb}}
+          as footnote.
+          Return value has type \type(Type.inline-boxes);.
+          Associate with a place inserted this return value, 
+          and calculate inserting the footnote at processing page-breaking.  
+          As inline-boxes feature itself is equivalent as \codem{inline-nil}.
+        }%
+
+      \val-explain?:(`hook-page-break`)({hook-page-break})%
+        (Type.(((record [({page-number}, int)] --> (point --> unit))) --> inline-boxes)) {
+          Use as \codem{hook-page-break \meta{f}}.
+          Return value is inline-boxes.
+          Features in typesetting is equivalent as \codem{inline-nil}.
+          But, a hooking function \codem{\meta{f}} starts
+          when fixed place including here is which page in processing page-breaking.
+          
+          The case mined multiple hooking functions at the same page,
+          the functions starts by page contents order from initial to last.
+          
+          where result of page-breaking, inline-boxes which mined *hook* function 
+          is included at page ${n}, and refer point is ${\tuple{|x|y|}} then,
+          \codem{\meta{f}} starts when it received 
+          \codem{(\| page-number = \meta{n}\; \|)} as 1st argument 
+          and 
+          \codem{(\meta{x}, \meta{y})} as 2nd argument.
+          You can invoke *side-effects* with these argument values
+          like to register cross-reference linking based on page number with \codem{\meta{f}} 's contents.
+      
+        }%
+
+      About thing “To evaluate graphics is triggerd at page-breaking” 
+      written at \ref-chapter(`graphics`); is 
+      as same as this \codem{hook-page-break}.
+      Functions that receives reffered point and returns graphics,
+      like 4th argument of \codem{inline-graphics}, and 
+      hooking functions by \codem{hook-page-break}
+      are start evaluations at the same time.
+    }
+  >
++section{Basis of classfile}<
+
+  +p{
+    You got matter about page-breaking process.
```

Comment:
"get matter about" is rarely used.   Simply use "grasp" or "understand".

Reply:
change the word to "grasped"